<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="7" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-18T07:03:22 EDT" name="EmployeesRestAPISuite" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2901">
    <groups>
    </groups>
    <test started-at="2021-10-18T07:03:22 EDT" name="EmployeesRestAPITest" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2901">
      <class name="com.employeeapi.testcase.TC_001_GET_ALL_EMP">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:22 EDT" name="setup" finished-at="2021-10-18T07:03:22 EDT" duration-ms="43" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="getAllEmpData()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:22 EDT" name="getAllEmpData" finished-at="2021-10-18T07:03:25 EDT" depends-on-methods="com.employeeapi.base.TestBase.setup" duration-ms="2574" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAllEmpData -->
        <test-method signature="validationBody()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationBody" finished-at="2021-10-18T07:03:25 EDT" duration-ms="18" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationBody -->
        <test-method signature="validationCode()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationCode" finished-at="2021-10-18T07:03:25 EDT" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[-->FAIL<-- statusCodeAcutal 200 expected [201] but found [200]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: -->FAIL<-- statusCodeAcutal 200 expected [201] but found [200]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at com.employeeapi.testcase.Validation.validationStatusCode(Validation.java:38)
at com.employeeapi.testcase.TC_001_GET_ALL_EMP.validationCode(TC_001_GET_ALL_EMP.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationCode -->
        <test-method signature="validationContent()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationContent" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationContent -->
        <test-method signature="validationEncoding()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationEncoding" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationEncoding -->
        <test-method signature="validationLength()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationLength" finished-at="2021-10-18T07:03:25 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationLength -->
        <test-method signature="validationLine()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationLine" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationLine -->
        <test-method signature="validationServer()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationServer" finished-at="2021-10-18T07:03:25 EDT" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationServer -->
        <test-method signature="validationTime()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="validationTime" finished-at="2021-10-18T07:03:25 EDT" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationTime -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.employeeapi.testcase.TC_001_GET_ALL_EMP@77847718]" started-at="2021-10-18T07:03:25 EDT" name="tearDown" finished-at="2021-10-18T07:03:25 EDT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.employeeapi.testcase.TC_001_GET_ALL_EMP -->
    </test> <!-- EmployeesRestAPITest -->
  </suite> <!-- EmployeesRestAPISuite -->
</testng-results>
